ERROR - 2021-11-16 06:02:48 --> Error connecting to the database: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
CRITICAL - 2021-11-16 06:02:48 --> Unable to connect to the database.
Main connection [MySQLi]: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
#0 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseConnection.php(625): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `po...', Array, false)
#2 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#3 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#4 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(687): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\app\Controllers\Blog.php(52): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(928): App\Controllers\Blog->new()
#8 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Blog))
#9 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#12 {main}
ERROR - 2021-11-16 06:07:55 --> Error connecting to the database: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
CRITICAL - 2021-11-16 06:07:55 --> Unable to connect to the database.
Main connection [MySQLi]: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
#0 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseConnection.php(625): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `po...', Array, false)
#2 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#3 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#4 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(687): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\app\Controllers\Blog.php(52): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(928): App\Controllers\Blog->new()
#8 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Blog))
#9 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#12 {main}
ERROR - 2021-11-16 06:10:51 --> Error connecting to the database: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
CRITICAL - 2021-11-16 06:10:51 --> Unable to connect to the database.
Main connection [MySQLi]: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
#0 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseConnection.php(625): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `po...', Array, false)
#2 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#3 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#4 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(687): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\app\Controllers\Blog.php(52): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(928): App\Controllers\Blog->new()
#8 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Blog))
#9 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2021-11-16 06:20:31 --> Undefined property: App\Controllers\Blog::$db
#0 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\app\Controllers\Blog.php(47): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined prope...', 'C:\\xampp\\htdocs...', 47)
#1 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(928): App\Controllers\Blog->new()
#2 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Blog))
#3 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-11-16 06:22:24 --> Undefined property: App\Controllers\Blog::$db
#0 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\app\Controllers\Blog.php(50): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined prope...', 'C:\\xampp\\htdocs...', 50)
#1 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(928): App\Controllers\Blog->new()
#2 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Blog))
#3 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-11-16 06:22:32 --> Undefined property: App\Controllers\Blog::$db
#0 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\app\Controllers\Blog.php(50): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined prope...', 'C:\\xampp\\htdocs...', 50)
#1 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(928): App\Controllers\Blog->new()
#2 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Blog))
#3 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-11-16 06:22:53 --> Undefined property: App\Controllers\Blog::$db
#0 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\app\Controllers\Blog.php(51): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined prope...', 'C:\\xampp\\htdocs...', 51)
#1 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(928): App\Controllers\Blog->new()
#2 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Blog))
#3 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-11-16 06:23:16 --> Undefined property: App\Controllers\Blog::$db
#0 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\app\Controllers\Blog.php(52): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined prope...', 'C:\\xampp\\htdocs...', 52)
#1 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(928): App\Controllers\Blog->new()
#2 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Blog))
#3 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#6 {main}
ERROR - 2021-11-16 06:23:33 --> Error connecting to the database: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
CRITICAL - 2021-11-16 06:23:33 --> Unable to connect to the database.
Main connection [MySQLi]: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
#0 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseConnection.php(625): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `po...', Array, false)
#2 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#3 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#4 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(687): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\app\Controllers\Blog.php(60): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(928): App\Controllers\Blog->new()
#8 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Blog))
#9 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#12 {main}
ERROR - 2021-11-16 06:24:15 --> Error connecting to the database: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
CRITICAL - 2021-11-16 06:24:15 --> Unable to connect to the database.
Main connection [MySQLi]: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
#0 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseConnection.php(625): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `po...', Array, false)
#2 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#3 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#4 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(687): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\app\Controllers\Blog.php(60): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(928): App\Controllers\Blog->new()
#8 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Blog))
#9 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#12 {main}
ERROR - 2021-11-16 06:28:13 --> Error connecting to the database: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
CRITICAL - 2021-11-16 06:28:13 --> Unable to connect to the database.
Main connection [MySQLi]: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
#0 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseConnection.php(625): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `po...', Array, false)
#2 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#3 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#4 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(687): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\app\Controllers\Blog.php(60): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(928): App\Controllers\Blog->new()
#8 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Blog))
#9 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2021-11-16 08:35:23 --> print_r() expects at least 1 argument, 0 given
#0 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\app\Controllers\Dbs.php(12): print_r()
#1 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(928): App\Controllers\Dbs->index()
#2 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dbs))
#3 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-11-16 08:36:08 --> Call to undefined method App\Models\BlogModel::getMethod
#0 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\app\Controllers\Dbs.php(12): CodeIgniter\Model->__call('getMethod', Array)
#1 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(928): App\Controllers\Dbs->index()
#2 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dbs))
#3 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-11-16 08:40:07 --> Array to string conversion
#0 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\app\Controllers\Dbs.php(12): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Array to string...', 'C:\\xampp\\htdocs...', 12)
#1 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(928): App\Controllers\Dbs->index()
#2 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dbs))
#3 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#6 {main}
ERROR - 2021-11-16 08:41:27 --> Error connecting to the database: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
CRITICAL - 2021-11-16 08:41:27 --> Unable to connect to the database.
Main connection [MySQLi]: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
#0 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseConnection.php(625): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `po...', Array, false)
#2 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#3 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#4 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(687): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\app\Controllers\Blog.php(61): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(928): App\Controllers\Blog->new()
#8 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Blog))
#9 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#12 {main}
ERROR - 2021-11-16 08:42:21 --> Error connecting to the database: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
CRITICAL - 2021-11-16 08:42:21 --> Unable to connect to the database.
Main connection [MySQLi]: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
#0 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseConnection.php(625): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `po...', Array, false)
#2 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#3 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#4 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(687): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\app\Controllers\Blog.php(61): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(928): App\Controllers\Blog->new()
#8 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Blog))
#9 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#12 {main}
ERROR - 2021-11-16 08:42:37 --> Error connecting to the database: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
CRITICAL - 2021-11-16 08:42:37 --> Unable to connect to the database.
Main connection [MySQLi]: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
#0 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseConnection.php(625): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `po...', Array, false)
#2 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#3 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#4 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(687): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\app\Controllers\Blog.php(61): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(928): App\Controllers\Blog->new()
#8 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Blog))
#9 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2021-11-16 08:44:55 --> Attempt to assign property "db" on null
#0 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(928): App\Controllers\Blog->new()
#1 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Blog))
#2 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#5 {main}
ERROR - 2021-11-16 09:20:17 --> Error connecting to the database: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
CRITICAL - 2021-11-16 09:20:17 --> Unable to connect to the database.
Main connection [MySQLi]: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
#0 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseConnection.php(625): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `po...', Array, false)
#2 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#3 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#4 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(687): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\app\Controllers\Blog.php(61): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(928): App\Controllers\Blog->new()
#8 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Blog))
#9 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#12 {main}
ERROR - 2021-11-16 09:21:14 --> Error connecting to the database: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
CRITICAL - 2021-11-16 09:21:14 --> Unable to connect to the database.
Main connection [MySQLi]: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
#0 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseConnection.php(625): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `po...', Array, false)
#2 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#3 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#4 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(687): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\app\Controllers\Blog.php(61): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(928): App\Controllers\Blog->new()
#8 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Blog))
#9 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#12 {main}
ERROR - 2021-11-16 09:23:51 --> Error connecting to the database: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
CRITICAL - 2021-11-16 09:23:51 --> Unable to connect to the database.
Main connection [MySQLi]: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
#0 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseConnection.php(625): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `po...', Array, false)
#2 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#3 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#4 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(687): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\app\Controllers\Blog.php(61): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(928): App\Controllers\Blog->new()
#8 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Blog))
#9 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#12 {main}
ERROR - 2021-11-16 09:24:00 --> Error connecting to the database: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
CRITICAL - 2021-11-16 09:24:00 --> Unable to connect to the database.
Main connection [MySQLi]: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
#0 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseConnection.php(625): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `po...', Array, false)
#2 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#3 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#4 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(687): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\app\Controllers\Blog.php(61): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(928): App\Controllers\Blog->new()
#8 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Blog))
#9 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#12 {main}
ERROR - 2021-11-16 09:25:23 --> Error connecting to the database: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
CRITICAL - 2021-11-16 09:25:23 --> Unable to connect to the database.
Main connection [MySQLi]: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
#0 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseConnection.php(625): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `po...', Array, false)
#2 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#3 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#4 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(687): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\app\Controllers\Blog.php(61): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(928): App\Controllers\Blog->new()
#8 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Blog))
#9 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#12 {main}
ERROR - 2021-11-16 09:25:30 --> Error connecting to the database: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
CRITICAL - 2021-11-16 09:25:30 --> Unable to connect to the database.
Main connection [MySQLi]: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
#0 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseConnection.php(625): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `po...', Array, false)
#2 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#3 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#4 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(687): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\app\Controllers\Blog.php(61): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(928): App\Controllers\Blog->new()
#8 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Blog))
#9 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#12 {main}
ERROR - 2021-11-16 09:25:43 --> Error connecting to the database: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
CRITICAL - 2021-11-16 09:25:43 --> Unable to connect to the database.
Main connection [MySQLi]: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
#0 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseConnection.php(625): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `po...', Array, false)
#2 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#3 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#4 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(687): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\app\Controllers\Blog.php(61): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(928): App\Controllers\Blog->new()
#8 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Blog))
#9 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#12 {main}
ERROR - 2021-11-16 09:26:20 --> Error connecting to the database: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
CRITICAL - 2021-11-16 09:26:20 --> Unable to connect to the database.
Main connection [MySQLi]: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
#0 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseConnection.php(625): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `po...', Array, false)
#2 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#3 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#4 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(687): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\app\Controllers\Blog.php(61): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(928): App\Controllers\Blog->new()
#8 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Blog))
#9 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#12 {main}
ERROR - 2021-11-16 09:26:35 --> Error connecting to the database: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
CRITICAL - 2021-11-16 09:26:35 --> Unable to connect to the database.
Main connection [MySQLi]: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
#0 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseConnection.php(625): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `po...', Array, false)
#2 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#3 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#4 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(687): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\app\Controllers\Blog.php(61): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(928): App\Controllers\Blog->new()
#8 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Blog))
#9 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2021-11-16 09:26:37 --> Undefined variable $title
#0 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\app\Views\new_post.php(7): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined varia...', 'C:\\xampp\\htdocs...', 7)
#1 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\View\View.php(230): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\View\View.php(232): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Common.php(1271): CodeIgniter\View\View->render('new_post', Array, true)
#4 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\app\Controllers\Blog.php(64): view('new_post')
#5 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(928): App\Controllers\Blog->new()
#6 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Blog))
#7 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#10 {main}
ERROR - 2021-11-16 09:29:14 --> Error connecting to the database: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
CRITICAL - 2021-11-16 09:29:14 --> Unable to connect to the database.
Main connection [MySQLi]: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
#0 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseConnection.php(625): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `po...', Array, false)
#2 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#3 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#4 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(687): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\app\Controllers\Blog.php(61): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(928): App\Controllers\Blog->new()
#8 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Blog))
#9 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#12 {main}
ERROR - 2021-11-16 09:32:38 --> Error connecting to the database: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
CRITICAL - 2021-11-16 09:32:38 --> Unable to connect to the database.
Main connection [MySQLi]: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
#0 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseConnection.php(625): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `po...', Array, false)
#2 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#3 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#4 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(687): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\app\Controllers\Blog.php(61): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(928): App\Controllers\Blog->new()
#8 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Blog))
#9 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2021-11-16 09:35:13 --> Undefined variable $default
#0 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\app\Controllers\Blog.php(57): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined varia...', 'C:\\xampp\\htdocs...', 57)
#1 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(928): App\Controllers\Blog->new()
#2 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Blog))
#3 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-11-16 09:36:15 --> $default is not a valid database connection group.
#0 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\app\Controllers\Blog.php(57): CodeIgniter\Database\Config::connect('$default')
#1 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(928): App\Controllers\Blog->new()
#2 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Blog))
#3 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#6 {main}
ERROR - 2021-11-16 09:36:27 --> Error connecting to the database: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
CRITICAL - 2021-11-16 09:36:27 --> Unable to connect to the database.
Main connection [MySQLi]: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
#0 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseConnection.php(625): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `po...', Array, false)
#2 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#3 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#4 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(687): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\app\Controllers\Blog.php(61): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(928): App\Controllers\Blog->new()
#8 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Blog))
#9 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2021-11-16 09:36:46 --> syntax error, unexpected token ")", expecting ":"
#0 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Router\Router.php(169): CodeIgniter\Router\Router->autoRoute('blog/new')
#1 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(809): CodeIgniter\Router\Router->handle('blog/new')
#2 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(389): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
#3 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#6 {main}
ERROR - 2021-11-16 09:37:00 --> Error connecting to the database: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
CRITICAL - 2021-11-16 09:37:00 --> Unable to connect to the database.
Main connection [MySQLi]: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
#0 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseConnection.php(625): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `po...', Array, false)
#2 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#3 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#4 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(687): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\app\Controllers\Blog.php(61): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(928): App\Controllers\Blog->new()
#8 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Blog))
#9 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2021-11-16 10:12:47 --> Invalid file: partials/sidebar.php
#0 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\View\View.php(213): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('partials/sideba...')
#1 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\View\View.php(487): CodeIgniter\View\View->render('partials/sideba...', NULL, true)
#2 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\app\Views\aplogin.php(27): CodeIgniter\View\View->include('partials/sideba...')
#3 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\View\View.php(230): include('C:\\xampp\\htdocs...')
#4 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\View\View.php(232): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#5 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Common.php(1271): CodeIgniter\View\View->render('aplogin', Array, true)
#6 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\app\Controllers\Cloth.php(12): view('aplogin')
#7 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(928): App\Controllers\Cloth->Login()
#8 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cloth))
#9 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2021-11-16 10:13:34 --> Invalid file: assets/aplogin.php
#0 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\View\View.php(213): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('assets/aplogin....')
#1 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\View\View.php(487): CodeIgniter\View\View->render('assets/aplogin', NULL, true)
#2 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\app\Views\aplogin.php(27): CodeIgniter\View\View->include('assets/aplogin')
#3 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\View\View.php(230): include('C:\\xampp\\htdocs...')
#4 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\View\View.php(232): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#5 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Common.php(1271): CodeIgniter\View\View->render('aplogin', Array, true)
#6 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\app\Controllers\Cloth.php(12): view('aplogin')
#7 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(928): App\Controllers\Cloth->Login()
#8 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cloth))
#9 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2021-11-16 10:13:43 --> Invalid file: assets/aplogin.php
#0 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\View\View.php(213): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('assets/aplogin....')
#1 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\View\View.php(487): CodeIgniter\View\View->render('assets/aplogin....', NULL, true)
#2 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\app\Views\aplogin.php(27): CodeIgniter\View\View->include('assets/aplogin....')
#3 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\View\View.php(230): include('C:\\xampp\\htdocs...')
#4 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\View\View.php(232): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#5 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Common.php(1271): CodeIgniter\View\View->render('aplogin', Array, true)
#6 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\app\Controllers\Cloth.php(12): view('aplogin')
#7 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(928): App\Controllers\Cloth->Login()
#8 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cloth))
#9 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2021-11-16 10:31:54 --> Too few arguments to function App\Controllers\Blog::Post(), 0 passed in C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php on line 928 and exactly 1 expected
#0 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(928): App\Controllers\Blog->Post()
#1 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Blog))
#2 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#5 {main}
CRITICAL - 2021-11-16 10:32:45 --> Too few arguments to function App\Controllers\Blog::Post(), 0 passed in C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php on line 928 and exactly 1 expected
#0 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(928): App\Controllers\Blog->Post()
#1 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Blog))
#2 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#5 {main}
CRITICAL - 2021-11-16 10:33:22 --> Too few arguments to function App\Controllers\Blog::Post(), 0 passed in C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php on line 928 and exactly 1 expected
#0 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(928): App\Controllers\Blog->Post()
#1 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Blog))
#2 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#5 {main}
CRITICAL - 2021-11-16 10:33:23 --> Too few arguments to function App\Controllers\Blog::Post(), 0 passed in C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php on line 928 and exactly 1 expected
#0 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(928): App\Controllers\Blog->Post()
#1 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Blog))
#2 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#5 {main}
CRITICAL - 2021-11-16 10:33:36 --> Too few arguments to function App\Controllers\Blog::Post(), 0 passed in C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php on line 928 and exactly 1 expected
#0 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(928): App\Controllers\Blog->Post()
#1 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Blog))
#2 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#5 {main}
ERROR - 2021-11-16 10:37:14 --> Error connecting to the database: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
CRITICAL - 2021-11-16 10:37:14 --> Unable to connect to the database.
Main connection [MySQLi]: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
#0 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseConnection.php(625): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `po...', Array, false)
#2 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#3 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#4 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(687): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\app\Controllers\Blog.php(71): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(928): App\Controllers\Blog->new()
#8 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Blog))
#9 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#12 {main}
ERROR - 2021-11-16 10:37:37 --> Error connecting to the database: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
CRITICAL - 2021-11-16 10:37:37 --> Unable to connect to the database.
Main connection [MySQLi]: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
#0 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseConnection.php(625): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `po...', Array, false)
#2 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#3 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#4 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(687): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\app\Controllers\Blog.php(71): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(928): App\Controllers\Blog->new()
#8 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Blog))
#9 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#12 {main}
ERROR - 2021-11-16 10:39:49 --> Error connecting to the database: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
CRITICAL - 2021-11-16 10:39:49 --> Unable to connect to the database.
Main connection [MySQLi]: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
#0 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseConnection.php(625): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `po...', Array, false)
#2 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#3 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#4 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(687): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\app\Controllers\Blog.php(71): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(928): App\Controllers\Blog->new()
#8 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Blog))
#9 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2021-11-16 10:44:16 --> Class "App\BaseModel\BlogModel" not found
#0 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(928): App\Controllers\Blog->new()
#1 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Blog))
#2 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#5 {main}
CRITICAL - 2021-11-16 10:44:56 --> Class App\Models\BlogModel contains 16 abstract methods and must therefore be declared abstract or implement the remaining methods (CodeIgniter\BaseModel::doFind, CodeIgniter\BaseModel::doFindColumn, CodeIgniter\BaseModel::doFindAll, ...)
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2021-11-16 10:45:57 --> Class App\BaseModel\BlogModel contains 16 abstract methods and must therefore be declared abstract or implement the remaining methods (CodeIgniter\BaseModel::doFind, CodeIgniter\BaseModel::doFindColumn, CodeIgniter\BaseModel::doFindAll, ...)
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
ERROR - 2021-11-16 10:46:11 --> Error connecting to the database: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
CRITICAL - 2021-11-16 10:46:11 --> Unable to connect to the database.
Main connection [MySQLi]: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
#0 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseConnection.php(625): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `po...', Array, false)
#2 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#3 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#4 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(687): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\app\Controllers\Blog.php(71): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(928): App\Controllers\Blog->new()
#8 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Blog))
#9 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#12 {main}
ERROR - 2021-11-16 10:56:16 --> Error connecting to the database: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
CRITICAL - 2021-11-16 10:56:16 --> Unable to connect to the database.
Main connection [MySQLi]: Call to undefined function CodeIgniter\Database\MySQLi\mysqli_init()
#0 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseConnection.php(625): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `po...', Array, false)
#2 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#3 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#4 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, false)
#5 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\BaseModel.php(687): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\app\Controllers\Blog.php(71): CodeIgniter\BaseModel->save(Array)
#7 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(928): App\Controllers\Blog->new()
#8 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Blog))
#9 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\CodeIgniter4\CodeIgniter4\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#12 {main}
